pipeline_dp/combiners.py

```diff
@@ -474,6 +494,9 @@ def compute_metrics(self, compound_accumulator: AccumulatorType):
     def metrics_names(self) -> List[str]:
         return self._metrics_to_compute
 
+    def explain_computation(self) -> str:
```

Comment:
This is probably List[Union[Callable, str]]?

Reply:
yeah, thanks